boards/arm/mimxrt1064_evk/doc/mimxrt1064_evk.rst

```diff
@@ -0,0 +1,197 @@
+.. _mimxrt1064_evk:
+
+NXP MIMXRT1064-EVK
+##################
+
+Overview
+********
+
+The i.MX RT1064 is the latest addition to the industry's first crossover
+processor series and expands the i.MX RT series to three scalable families.
+The i.MX RT1064 doubles the On-Chip SRAM to 1MB while keeping pin-to-pin
+compatibility with i.MX RT1050. This new series introduces additional features
+ideal for real-time applications such as High-Speed GPIO, CAN-FD, and
+synchronous parallel NAND/NOR/PSRAM controller. The i.MX RT1064 runs on the
+Arm® Cortex-M7® core at 600 MHz.
+
+.. image:: mimxrt1064_evk.jpg
```

Comment:
The image file in this PR is named ``mimxrt1064.jpg``, rename it or fix this reference.